ui/src/app/components/terms-of-service.tsx

```diff
@@ -62,6 +62,11 @@ export class TermsOfService extends React.Component<TosProps, TosState> {
       hasAckedAgreement: props.afterPrev,
       hasClickedNext: false,
     };
+    // This bypasses the scroll detection. A better implementation may be to provide a method
+    // on HtmlViewer to force the scroll, thus exercising the scroll detection.
+    (window as any).forceHasReadEntireAgreement = () => {
```

Comment:
I put on my hacker hat and tried to use this to let me continue on the TOS screen without doing a proper scroll.  It worked.

But do we care?  I think no.

Reply:
Indeed. We would want to be cautious around a captcha override, for example.